src/Symfony/Bundle/FrameworkBundle/Command/DotenvDumpCommand.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Dotenv\Dotenv;
+
+/**
+ * A console command to compile the contents of the .env* files into a PHP-optimized file called .env.local.php.
+ *
+ * @internal
+ */
+final class DotenvDumpCommand extends Command
```

Comment:
I just realized that this should be in the Dotenv component, isn't it?
Can you please move it there?

Reply:
But that would mean making the Dotenv component dependent on symfony/console. Would that be ok? Also, will everyone who uses Dotenv  have the need to export these variables in the same place?

Reply:
Optional feature don't need to be in "require". Putting "symfony/console" in "require-dev" is how we should do this.